src/directive.js

```diff
@@ -17,10 +17,11 @@
       },
       transclude: true,
       template: '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
-        '<input ng-focus="onInputFocus()" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
+        '<input ng-show="isEditing" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
```

Comment:
What was the motivation for this change? `onInputFocus()` ensures that all of the text in the input box is selected, particularly after a value has been entered and the input gets focus again.